src/warc2zim/url_rewriting.py

```diff
@@ -280,7 +285,10 @@ def __call__(self, item_url: str, *, rewrite_all_url: bool = True) -> str:
         if rewrite_all_url or item_path in self.existing_zim_paths:
             return self.get_document_uri(item_path, item_fragment)
         else:
-            if item_path not in self.missing_zim_paths:
+            if (
+                isinstance(self.missing_zim_paths, set)
```

Comment:
`self.missing_zim_paths is not None` is probably better than `instance(self.missing_zim_paths, set)`